packages/hoprd/crates/hopr-lib/src/lib.rs

```diff
@@ -206,10 +206,72 @@ mod native {
             self.aliases.read().await.clone()
         }
 
+        pub async fn get_balance(&self, balance_type: BalanceType) -> errors::Result<Balance> {
+            self.chain_api
```

Comment:
The question is whether to wrap this around the HoprChain, so that it offers a testable API, or do it like this, by dereferencing the internal objects.